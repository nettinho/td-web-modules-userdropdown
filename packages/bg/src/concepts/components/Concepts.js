import React from "react";
import { Header, Icon, Segment } from "semantic-ui-react";
import { FormattedMessage } from "react-intl";
import ConceptsActions from "./ConceptsActions";
import ConceptCrumbs from "./ConceptCrumbs";
import ConceptsLabelResults from "./ConceptsLabelResults";
import ConceptsPagination from "./ConceptsPagination";
import ConceptsSearch from "./ConceptsSearch";
import ConceptsTable from "./ConceptsTable";
import ConceptsTabs from "./ConceptsTabs";
import ConceptSelectedFilters from "./ConceptSelectedFilters";

export const Concepts = () => (
  <React.Fragment>
    <ConceptCrumbs />
    <Segment>
      <Header as="h2">
        <Icon circular name="tags" />
        <Header.Content>
          <FormattedMessage id="concepts.header" />
          <Header.Subheader>
            <FormattedMessage id="concepts.subheader" />
          </Header.Subheader>
        </Header.Content>
      </Header>
      <Segment attached="bottom">
        <ConceptsTabs />
        <ConceptsActions />
        <ConceptsSearch />
        <ConceptSelectedFilters />
        <ConceptsLabelResults />
        <ConceptsTable />
        <ConceptsPagination />
      </Segment>
    </Segment>
  </React.Fragment>
);

export default Concepts;
