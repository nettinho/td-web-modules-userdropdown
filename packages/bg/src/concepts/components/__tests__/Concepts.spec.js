import React from "react";
import { shallow } from "enzyme";
import { Concepts } from "../Concepts";

describe("<Concepts />", () => {
  it("matches the latest snapshot", () => {
    const concepts = [
      { id: 1, name: "s1", description: "dd", version: "vs", status: "st" }
    ];
    const props = { concepts };
    const wrapper = shallow(<Concepts {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
