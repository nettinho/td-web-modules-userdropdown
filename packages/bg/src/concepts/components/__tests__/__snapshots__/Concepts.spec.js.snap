// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Concepts /> matches the latest snapshot 1`] = `
<React.Fragment>
  <Connect(ConceptCrumbs) />
  <Segment>
    <Header
      as="h2"
    >
      <Icon
        as="i"
        circular={true}
        name="tags"
      />
      <HeaderContent>
        <FormattedMessage
          id="concepts.header"
          values={Object {}}
        />
        <HeaderSubheader>
          <FormattedMessage
            id="concepts.subheader"
            values={Object {}}
          />
        </HeaderSubheader>
      </HeaderContent>
    </Header>
    <Segment
      attached="bottom"
    >
      <withRouter(InjectIntl(Connect(ConceptTabs))) />
      <Connect(ConceptsActions) />
      <InjectIntl(Connect(ConceptsSearch)) />
      <InjectIntl(Connect(ConceptSelectedFilters)) />
      <InjectIntl(Connect(ConceptsLabelResults)) />
      <withRouter(Connect(ConceptsTable)) />
      <Connect(ConceptsPagination) />
    </Segment>
  </Segment>
</React.Fragment>
`;
